packages/playground/src/components/k8s_deployment_table.vue

```diff
@@ -58,7 +58,7 @@
 
     <AccessDeploymentAlert />
 
-    <InputTooltip tooltip="List all deployments, including those created outside the Dashboard." inline>
+    <InputTooltip tooltip="Didn't find your deployments in the list? Toggle to show all deployments." inline>
```

Comment:
In case the switch is on, the tooltip wouldn't be accurate. It needs to be all-inclusive.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/708dc106-712e-4cc5-b65d-b14cfb6bbb63)

Reply:
do you have any suggestions?

Reply:
I'm not sure tbh but maybe something like "Toggle to show/hide deployments"?

Reply:
Another Option: you can also add a v-if to the tooltip such as ``<InputTooltip tooltip="Didn't find your deployments in the list? Toggle to show all deployments." inline v-if="!showAllDeployment" >`` so that the tooltip does not show when the switch is flipped on.

Or you can create separate tooltips for when it is switched on/off

Reply:
or "Toggle to show/hide deployments created outside the Dashboard."

Reply:
As per discussion with @AhmedHanafy725, the tooltip text originally written by samar is alright